### PR TITLE
Health Gateway Issuer

### PR DESCRIPTION
**Pull request by WadeBarnes**
Addresses https://github.com/bcgov/von/issues/373

### CODE REVIEW

**Comment by WadeBarnes** [comment]
Other then turning off read-only ledger for `prod` once the agent is registered, this is read to go.  I can make that update directly once complete, so it doe not need to hold up this PR.

File: openshift/manage [hunk 1]
```diff
@@ -15,25 +44,36 @@ usage () {
...
-          $0 -p a2a -e dev accepttaa https://toip-vc-authn-controller-dev.apps.silver.devops.gov.bc.ca
+          $0 -p a2a -e dev accepttaa https://agent-admin-url-dev.apps.silver.devops.gov.bc.ca
+          $0 -p phg -e dev accepttaa https://public-health-gateway-agent-admin-dev.apps.silver.devops.gov.bc.ca agent-public-health-gateway
+
+    registerSovrin <baseUrl> <stagingnet|buildernet> [secretName] [secretKey] [namespace]
+      - Register the agent on the target ledger as an ENDORSER.
+
+        Examples;
+          $0 -p default -e dev registerSovrin https://agent-admin-url-dev.apps.silver.devops.gov.bc.ca stagingnet
+          $0 -p phg -e dev registerSovrin https://public-health-gateway-agent-admin-dev.apps.silver.devops.gov.bc.ca StagingNet agent-public-health-gateway
```
Comment (esune) [inline openshift/manage:67]: Nit.: stagingnet is case-sensitive, isn't it? It should be all lowercase, I believe.

**Review by esune — CHANGES_REQUESTED** [review]
Only a detail about the string case in the example, looks great otherwise!

File: openshift/manage (hunk 1 above)
Comment (WadeBarnes) [reply openshift/manage]: It's set `tolower`.

**Review by esune — APPROVED** [review]

**Pull request merged**